src/components/admin/personen/PersonenInfoChange.spec.ts

```diff
@@ -0,0 +1,85 @@
+import { describe, it, expect, beforeEach, vi, type Mock } from 'vitest';
+import { mount, VueWrapper } from '@vue/test-utils';
+import PersonInfoChange from './PersonenInfoChange.vue';
+
+describe('PersonInfoChange', () => {
+  let wrapper: VueWrapper;
+  const mockConfirmUnsavedChangesAction: Mock = vi.fn();
+
+  beforeEach(() => {
+    wrapper = mount(PersonInfoChange, {
+      props: {
+        confirmUnsavedChangesAction: mockConfirmUnsavedChangesAction,
+        hasKopersRolle: true,
+        selectedKopersNrPersonInfoProps: {
+          error: false,
+          'error-messages': [],
+          onBlur: () => vi.fn(),
+          onChange: () => vi.fn(),
+          onInput: () => vi.fn(),
+        },
+        selectedKopersNrPersonInfo: '123456',
+        selectedVornameProps: {
+          error: false,
+          'error-messages': [],
+          onBlur: () => vi.fn(),
+          onChange: () => vi.fn(),
+          onInput: () => vi.fn(),
+        },
+        selectedVorname: 'John',
+        selectedFamiliennameProps: {
+          error: false,
+          'error-messages': [],
+          onBlur: () => vi.fn(),
+          onChange: () => vi.fn(),
+          onInput: () => vi.fn(),
+        },
+        selectedFamilienname: 'Doe',
+        showUnsavedChangesDialog: false,
+      },
+      global: {
+        mocks: {
+          $t: (key: string) => key, // Mock translation function
+        },
+      },
+    });
+  });
+
+  it('renders Vorname input', () => {
+    expect(wrapper.find('[data-testid="vorname-input"]').exists()).toBe(true);
+  });
+
+  it('renders Familienname input', () => {
+    expect(wrapper.find('[data-testid="familienname-input"]').exists()).toBe(true);
+  });
+
+  it('renders KopersInput when hasKopersRolle is true', () => {
+    expect(wrapper.findComponent({ name: 'KopersInput' }).exists()).toBe(true);
+  });
+
+  it('does not render KopersInput when hasKopersRolle is false', async () => {
+    await wrapper.setProps({ hasKopersRolle: false });
+    expect(wrapper.findComponent({ name: 'KopersInput' }).exists()).toBe(false);
+  });
+
+  it('emits update:selectedVorname when Vorname input changes', async () => {
+    const input: VueWrapper = wrapper.findComponent({ ref: 'vorname-input' });
+    await input.setValue('Jane');
+    expect(wrapper.emitted('update:selectedVorname')).toBeTruthy();
+    expect(wrapper.emitted('update:selectedVorname')?.[0]).toEqual(['Jane']);
```

Comment:
```suggestion
    expect(wrapper.emitted('update:selectedVorname')?[0]).toEqual(['Jane']);
```
Ist der Punkt hier und weiter unten notwendig?